ext/search_kit/ang/crmSearchDisplayBatch/crmSearchDisplayBatch.component.js

```diff
@@ -57,14 +57,14 @@
           this.reportLinks = [
             {
               title: ts('View My Import Batches'),
-              href: CRM.url('civicrm/imports/my-listing'),
+              href: CRM.url('civicrm/imports/my-listing#/?job_type=search_batch_import'),
```

Comment:
CRM.url doesn't work if you do CRM.url('civicrm/imports/my-listing#/', {job_type: 'search_batch_import'}) because it splits the url at ```#``` and adds that [back at the end](https://github.com/civicrm/civicrm-core/pull/21020/files).

Reply:
Yea it's meant for working with server-side variables

Reply:
@larssandergreen I guess we could do something really clever and support a syntax like 
```js
CRM.url('civicrm/imports/my-listing', {'#job_type': 'search_batch_import'})
```